Bio/PDB/PDBList.py

```diff
@@ -435,52 +443,62 @@ def update_pdb(self, file_format=None):
         automatically downloads the according PDB files.
         You can call this module as a weekly cron job.
         """
-        if not os.path.isdir(self.local_pdb):
-            raise PDBListError(
-                f"Local PDB diretory does not exist (path: {self.local_pdb})."
-            )
-        if not os.path.isdir(self.obsolete_pdb):
-            raise PDBListError(
-                f"Obsolete PDB diretory does not exist (path: {self.obsolete_pdb})."
-            )
-
-        # Deprecation warning
-        file_format = self._print_default_format_warning(file_format)
+        if isinstance(file_format, FileFormat):
+            file_format_enum = file_format
+        else:
+            file_format = self._print_default_format_warning(file_format)
+            file_format_enum = FileFormat.from_label(file_format)
 
         new, modified, obsolete = self.get_recent_changes()
-
         for pdb_code in new + modified:
             try:
-                self.retrieve_pdb_file(pdb_code, file_format=file_format)
-            except Exception:
-                print(f"error {pdb_code}\n")
-                # you can insert here some more log notes that
-                # something has gone wrong.
+                self.retrieve_pdb_file(
+                    pdb_code,
+                    file_format=file_format_enum,
+                    overwrite=pdb_code in modified,
```

Comment:
Notice that the PDB files tagged as `modified` were not overridden.
Regarding the context, I think it's what we want, wdyt ?